webdriver/tests/bidi/script/add_preload_script/add_preload_script.py

```diff
@@ -90,14 +90,18 @@ async def on_event(method, data):
 
 
 @pytest.mark.asyncio
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
 async def test_add_preload_script_in_iframe(
```

Comment:
Isn't that a dupe of the test in `contexts.py`?

Reply:
This one does not specify the contexts.

Reply:
Yes, but it also doesn't add any more coverage. The point of the test for same/cross origin is that we test that the preload script gets correctly setup in the appropriate frame as well. So whether we pass in the context or not, what does it make a difference for that scenario?